src/main/java/build/buf/protovalidate/NowVariable.java

```diff
@@ -14,41 +14,39 @@
 
 package build.buf.protovalidate;
 
+import com.google.protobuf.Timestamp;
+import dev.cel.runtime.CelVariableResolver;
 import java.time.Instant;
+import java.util.Optional;
 import org.jspecify.annotations.Nullable;
-import org.projectnessie.cel.common.types.TimestampT;
-import org.projectnessie.cel.interpreter.Activation;
-import org.projectnessie.cel.interpreter.ResolvedValue;
 
 /**
- * {@link NowVariable} implements {@link Activation}, providing a lazily produced timestamp for
- * accessing the variable `now` that's constant within an evaluation.
+ * {@link NowVariable} implements {@link CelVariableResolver}, providing a lazily produced timestamp
+ * for accessing the variable `now` that's constant within an evaluation.
  */
-class NowVariable implements Activation {
+class NowVariable implements CelVariableResolver {
```

Comment:
IIRC, we had a bug early on where `now` was cached for a validator instance, instead of for a validation pass. The conformance tests can't catch it. Would be good to be sure we don't regress here.

Reply:
I did not know that. I checked the references for `NowVariable` class, it is only created [here](https://github.com/bufbuild/protovalidate-java/blob/e13ebd082b058738481a9c9c517d1dd7d0aeb5ee/src/main/java/build/buf/protovalidate/Variable.java#L58) alongside `this` variable.